### PR TITLE
added tests to search products on any org

### PR DESCRIPTION
**Pull request by ColeHiggins2**
When trying to access the products page and "Any Organization" is selected, Users should see a landing page that asks them to select an Org. 

Currently, there are no views in airgun that allow us to see this. This is the current workaround.

Link to airgun page for possible functionality https://github.com/SatelliteQE/airgun/pull/640
Screenshot from 2021-12-01 15-09-09

### CODE REVIEW

File: tests/foreman/ui/test_repository.py [hunk 1]
```diff
@@ -1062,3 +1063,35 @@ def test_positive_rpm_metadata_display():
...
+    :BZ: 1861970
+
+    :Steps:
+        1. Create an Organization and product
+        2. Set Organization to Any Organization
+        3. Navigate to Content -> products page
+        4. Assert that user must select and Organization
+
+    :expectedresults: UI asks to select an Organization on products page
+    when Any Organization is selected
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_repository.py:1085]:
```suggestion
    :expectedresults: UI asks to select an Organization on products page
        when Any Organization is selected
```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_repository.py [hunk 2]
```diff
@@ -1062,3 +1063,35 @@ def test_positive_rpm_metadata_display():
...
+
+    :customerscenario: true
+
+    :BZ: 1861970
+
+    :Steps:
+        1. Create an Organization and product
+        2. Set Organization to Any Organization
+        3. Navigate to Content -> products page
+        4. Assert that user must select and Organization
```
Comment (swadeley) [inline tests/foreman/ui/test_repository.py:1082]:
```suggestion
        4. Assert that user must select an Organization
```

**Comment by latran** [comment]
@ColeHiggins2 any updates on this?  You still need help?

**Comment by JacobCallahan** [comment]
Bump

**Comment by Gauravtalreja1** [comment]
@ColeHiggins2 Can you update this PR?

**Pull request closed without merging**